melodic/distribution.yaml

```
@@ -149,6 +149,35 @@ repositories:
       url: https://github.com/ubi-agni/agni_tf_tools.git
       version: master
     status: maintained
+  agxrobots_test:
+    doc:
+      type: git
+      url: https://github.com/885288421/agxrobots_test.git
+      version: 0.0.3
+    release:
+      packages:
+      - async_port
+      - hunter_base
+      - hunter_bringup
+      - hunter_msgs
+      - scout_base
+      - scout_bringup
+      - scout_msgs
+      - tracer_base
+      - tracer_bringup
+      - tracer_description
+      - tracer_gazebo_sim
+      - tracer_msgs
+      - ugv_sdk
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/885288421/agxrobots_test_release.git
+      version: 0.0.3-8
+    source:
+      type: git
+      url: https://github.com/885288421/agxrobots_test.git
+      version: 0.0.3
```

```suggestion
      version: master
```
This should be a branch name.